terraform/010-cluster/main.tf

```diff
@@ -40,19 +40,36 @@ data "aws_acm_certificate" "wildcard" {
 }
 
 /*
- * Create application load balancer
+ * Create application load balancer for public access
  */
 module "alb" {
```

Comment:
You refactored outputs to use module named `external_alb` but the module here is still named `alb`.